packages/@aws-cdk-testing/framework-integ/README.md

```diff
@@ -46,3 +46,14 @@ export CERT_ARN=your_certificate_arn
 ```console
 yarn integ --disable-update-workflow aws-certificatemanager/test/integ.certificate-name
 ```
+
+### Error: Docker pull from `public.ecr.aws` results in "403 Forbidden"
+
+When running CDK integration tests locally, you may encounter a **"403 Forbidden"** error while trying to pull Docker images from `public.ecr.aws`.
+
+This happens because Docker requires proper authentication to access ECR repositories, even public ones.  
+You can fix it by authenticating with AWS ECR Public before running your tests:
+
+```sh
+aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
```

Comment:
Looks fine to me - let's link the [documentation](https://docs.aws.amazon.com/AmazonECR/latest/public/public-registry-auth.html#public-registry-auth-token) here as well, so that users can gather any relevant context + see the latest information in case this command ever changes.